daisy_workflows/image_build/enterprise_linux/kickstart/el6-post.cfg

```diff
@@ -186,4 +186,6 @@ echo "(hd0) /dev/sda" > /boot/grub/device.map
 echo -e "root (hd0,0) \n setup (hd0)" | grub \
   --config-file /boot/grub/grub.conf --batch \
   --device-map /boot/grub/device.map
+
+) 2>&1 | /usr/bin/tee /dev/ttyS0
```

Comment:
So, its a combination of things. Moving to cmdline instead of text makes the output usable- redirecting tty3 (which is the live output in anaconda) to ttyS0 means its all logged to the serial console and therefore saved. Some of the commands that we catch are because there are conditions where they can fail and its ok. (I'll add comments)

Reply:
this doesn't redirect tty3, it redirects TO tty3. regardless of the tty3 lines at top, you are only getting serial port output from the `tee` command at the end. `exec > /dev/ttyS0` would have eliminated the need for the subshell and the tee command.

Reply:
OH! Now I see why they are doing this. You're right, we only care about redirecting tty3 to ttyS0. These solutions all talk about getting live build logs to a screen and a log file. We don't actually care about that because we basically use the console as the log. So yeah- we can do this much easier.